docs/policy-provider.md

```diff
@@ -0,0 +1,50 @@
+# Policy Providers
+
+Ratify implements an extensible policy provider interface allowing for different policy providers to be created and registered. The policy provider to be used is determined by the policy plugin specified in the `policies` section of the configuration. 
+
+Currently, Ratify supports a Configuration based Policy Provider named `configPolicy`.
+
+## Sample Config
```

Comment:
Should this be documentation for default configuration

Reply:
I think we might need to discuss what we want the default config to be now. Since we have default policy support of 'all'. we can technically omit the `artifactTypePolicies` section entirely. So config would be:  
``` ...
    "policies": {
        "version": "1.0.0",
        "plugin": {
            "name": "configPolicy"
        }
    },
    ...
```

Reply:
ah I see. In the examples, we can still specify them for readability, but call out they can be omitted as they are the default. Can we also add a note about if there are artifacts that cannot be verified, how does it impact the overall result?